diff -uraN tuned-2.19.0/tuned.service omv-tuned-2.19.0/tuned.service
--- tuned-2.19.0/tuned.service	2022-08-19 20:28:45.000000000 +0200
+++ omv-tuned-2.19.0/tuned.service	2022-08-21 20:13:37.654175760 +0200
@@ -4,6 +4,7 @@
 Requires=dbus.service polkit.service
 Conflicts=cpupower.service auto-cpufreq.service tlp.service power-profiles-daemon.service
 Documentation=man:tuned(8) man:tuned.conf(5) man:tuned-adm(8)
+ConditionVirtualization=no
 
 [Service]
 Type=dbus
