diff -Naur tuned-2.11.0/tuned.service tuned-2.11.0.tpg/tuned.service
--- tuned-2.11.0/tuned.service	2019-03-21 22:10:46.000000000 +0000
+++ tuned-2.11.0.tpg/tuned.service	2019-03-22 07:52:57.625000000 +0000
@@ -4,6 +4,7 @@
 Requires=dbus.service polkit.service
 Conflicts=cpupower.service
 Documentation=man:tuned(8) man:tuned.conf(5) man:tuned-adm(8)
+ConditionVirtualization=no
 
 [Service]
 Type=dbus
